tests_python/test_debugger_json.py

```diff
@@ -4640,7 +4666,7 @@ def test_case_django_no_attribute_exception_breakpoint(case_setup_django_dap, jm
                     "protected": "inline",
                 },
             )
-            json_facade.write_set_exception_breakpoints(["raised"])
+            json_facade.write_set_exception_breakpoints(["raised", "uncaught"])
```

Comment:
This reproduces the django exception that I added the 'is_unwind' parameter for.